test.js

```diff
@@ -118,6 +119,38 @@ if (ci) {
       t.is(branch, real_branch);
     }
   });
+} else if (ci && platform === "gitlab") {
+  test("ci is correctly set", t => {
```

Comment:
shouldn't we check for `ci`?
It seems it is obvious that if `ci` is `gitlab` then the `platform` is also `gitlab`.  @swapnilmishra can you confirm? if so then IMO we should put the check for `ci`

Reply:
@kuldeepkeshwar 
Yes, that is true.
In that case, the condition will become `else if (ci === "gitlab") {` and then `t.is(ci, "gitlab");` test can be removed.

The reason I kept it like that was, to be in line with the other tests but can make this change.

Reply:
@swapnilmishra can you remove the `platform` check?
By removing the `platform` check test can fit the existing structure(get rid of if/else block).

Reply:
@kuldeepkeshwar 
Maybe I understood it wrong but if we remove `platform` check and also remove `if else` condition how will we know when to run `gitlab` tests(which can only be run on gitlab) and when to just run `github` tests(which can only be run on github).

I can quickly make the changes if you can help me figuring out how do you think the conditions would look like. Right now it is:

```js
if (ci && platform === "github") {
 // github tests
} else if (ci && platform === "gitlab") {
 // gitlab tests
}
```

Now If we remove the platform and remove any if-else conditions both `gitlab` and `github` tests would be running.

Reply:
I was thinking something like 👇

```js
const test = require("ava");

const {
  buildUrl,
  jobUrl,
  repo,
  sha,
  event,
  commit_message,
  pull_request_number,
  branch,
  ci
} = require("./index");

if (ci) {
  console.log("values: ", {
    repo,
    sha,
    event,
    commit_message,
    pull_request_number,
    branch,
    ci
  });
 // ... e.g
  test("sha is set", t => {
    const real_sha =
      process.env.TRAVIS_PULL_REQUEST_SHA ||
      process.env.TRAVIS_COMMIT ||
      process.env.CIRCLE_SHA1 ||
      process.env.WERCKER_GIT_COMMIT ||
      process.env.DRONE_COMMIT ||
      process.env.GITHUB_SHA|| 
      process.env.CI_COMMIT_SHA; // add env variable here

    t.is(sha, real_sha);
  });

} else if (ci && platform === "gitlab") { // ... remove this block
  
} else {
  test.skip("These tests can only run in CI environments", t => t.pass());
}
```
this is how existing test are structured currently.

Reply:
@kuldeepkeshwar 
Thanks for the explanation. I have made those changes now.